internal/pkg/cli/env_deploy_test.go

```diff
@@ -170,6 +172,14 @@ func TestDeployEnvOpts_Execute(t *testing.T) {
 			},
 			wantedErr: errors.New(`unmarshal environment manifest for "mockEnv"`),
 		},
+		"fail to validate cdn": {
+			setUpMocks: func(m *deployEnvExecuteMocks) {
+				m.ws.EXPECT().ReadEnvironmentManifest(gomock.Any()).Return([]byte("name: mockEnv\ntype: Environment\ncdn: true\n"), nil)
+				m.interpolator.EXPECT().Interpolate(gomock.Any()).Return("name: mockEnv\ntype: Environment\ncdn: true\n", nil)
+				m.describer.EXPECT().ValidateCFServiceDomainAliases().Return(fmt.Errorf("all services deployed in an environment with CloudFront enabled must have http.alias specified"))
```

Comment:
nit: since you're just testing a fake error anyways, might be easier to have something shorter
```suggestion
				m.describer.EXPECT().ValidateCFServiceDomainAliases().Return(errors.New("mock error"))
```